### DISCUSSION TITLE
Calculations beyond budgets

### DISCUSSION THREAD

**Discussion by ZeroIchiro** [kimai/kimai#4380] [Questions & Answers]
Dear community,

I was wondering the following.
I use Kimai to track and invoice my freelance I work for different customers.
I recently came up with the idea of "budgets" that customers can buy for a monthly fee. They thereby "buy" a certain amount of time (say 10h) they can spend on everything from teaching them how to use a certain program, to maintenance work, anything related to their IT or mobile devices, and afterwards, after these 10h are spent, each additional hour is calculated and billed extra.

What do I have to do to reflect this "agreement" in the "Activities" tab in Kimai Cloud?
Thank you very much in advance.
Kind Regards

Sascha Schroeder

**Comment by kevinpapst** [comment]
Add a new activity for every bought package with a time budget of 10 hours.
Once full, close the activity and book your times on the general "support" activity.
When the customer buys a new package, create a new activity with 10 hours budget.
Something like that.
